index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile_website</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <div class="wrapper">
+    <div class="wrapper-1">
+      <header>
+        <div class="logo"><a href="">My Logo</a></div>
+        <nav>
+          <a href="">
+            <img id="menu" src="./images/menu.png" alt="menu">
+          </a>
+
+        </nav>
+
+      </header>
+      
+    </div>
+    <section class="content">
+      <h1>I’m Kaweesi  Glad to see you!</h1>
+      <p>
+        I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <a id="connect" href="">LET’S CONNECT</a>
+      <ul class="content-list">
+        <li>
+          <a href=""><img src="./images/twitter.png" alt="twitter"></a>
```

Comment:
- [ ] Please reduce this white space between the header and hero section and make sure your design matches with the Figma design.
![image](https://user-images.githubusercontent.com/83803414/185658620-032f72b2-1b3d-4ff5-93a6-8b76772e8e29.png)